spec/requests/api/alerts_spec.rb

```diff
@@ -0,0 +1,48 @@
+describe "Alerts API" do
+  let(:alert_status)  { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_status2) { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_url) { alerts_url(alert_status.id) }
+  let(:vm) { FactoryGirl.create(:vm) }
+  let(:alert_definition) { FactoryGirl.create(:miq_alert, :severity => "info") }
+  let(:user) { FactoryGirl.create(:user_with_group) }
+
+  it "forbids access to actions without an appropriate role" do
+    api_basic_authorize
+    alert_status
+    alert_status2
+    run_get(alerts_url)
```

Comment:
no need for the above 2 lines, the alerts_url will instantiate the alert_status which is enough for this test.